locales/locales.php

```diff
@@ -961,15 +961,15 @@ public function __construct() {
 		$gu->google_code = 'gu';
 		$gu->facebook_locale = 'gu_IN';
 
-		$ha = new GP_Locale();
-		$ha->english_name = 'Hausa';
-		$ha->native_name = 'هَوُسَ';
-		$ha->lang_code_iso_639_1 = 'ha';
-		$ha->lang_code_iso_639_2 = 'hau';
-		$ha->slug = 'ha';
-		$ha->text_direction = 'rtl';
-		$ha->google_code = 'ha';
-		$ha->facebook_locale = 'ha_NG';
+		$hau = new GP_Locale();
+		$hau->english_name = 'Hausa';
+		$hau->native_name = 'Harshen Hausa';
+		$hau->lang_code_iso_639_1 = 'ha';
+		$hau->lang_code_iso_639_2 = 'hau';
+		$hau->lang_code_iso_639_3 = 'hau';
+		$hau->slug = 'hau';
```

Comment:
We're using ISO 639-3 codes for new WordPress locales and where possible adapting old ones as well (https://make.wordpress.org/polyglots/handbook/translating/requesting-a-new-locale/#find-your-language-code). Hausa has existed in GlotPress for a while but it doesn't have a WP locale and didn't have a site. The ISO 639-3 code is applied to the slugs as well as the language code for consistency. The site is already created with the slug `hau`.